### DISCUSSION TITLE
Edge Add-ons Publish API is now live!

### DISCUSSION THREAD

**Discussion by nagachaitanyalokam** [microsoft/MicrosoftEdge-Extensions#13] [Announcements]
With this addition, you can integrate APIs directly into your build pipeline and deploy packages and publish updated versions without visiting the Partner Center.  

To learn more, you can check the below documentations: 

[Using the Microsoft Edge Add-ons API](https://aka.ms/Using_MSFTEdgeAddons_API) 
[Microsoft Edge Add-ons API Reference](https://aka.ms/MSFTEdgeAddons_API_Reference)

Read more about changes made to streamline the process of submitting a Microsoft Edge extension [here](https://techcommunity.microsoft.com/t5/articles/what-s-new-on-microsoft-partner-center-for-edge-add-on/m-p/3270268)

**Comment by joelataccelm** [comment]
Why does It still take 10 days to publish an extension? Wasn't V3 supposed to speed this up? You're killing me.

**Reply by Jaifroid** [reply]
This is also an issue for me. The same MV3 extension in the Chrome Store is published in ~24 hours. Updates to the UWP app version in the Microsoft Store are also generally published in 24-48 hours. Update in WinGet takes 1-2 working days. Update in Mozilla Store is instant (subject to post-publication review). **_Only the Edge Extension Store takes literally days and days._**

**Reply by Gurjeet-msft** [reply]
Hello @joelataccelm 

Apologies for all the inconvenience caused. The review times for Edge extension submissions has improved in the recent times. I request you to please provide us more details of your extension and the submission you are referring to @[ext_dev_support@microsoft.com]

**Reply by joelataccelm** [reply]
@Gurjeet-msft I've done that, and It's not helpful. 7 full business days is apparently considered an improvement. 

It would be nice to hear why the process takes so long. How long is the extension sitting in a queue waiting for a reviewer to get to it? If it's just sitting there why can't we replace it with a new version and keep our place in line?

**Reply by vanowm** [reply]
Not only it takes up to 10 days for review, the whole "partner" website feels like running on machines from 1998 with dial-up connection. On top of that we also can't update addon's details while it's still waiting for an approval. How is this even considered as acceptable?
At this point I don't even know why I'm wasting my time with MS store at all. Even though the last point is the same for Chrome store, at least my addon being approved in nearly minutes there, so that's not an issue.
